tests/Unit/Helpers/DataStructures/Tensor/Expressions/EvaluateRank0TestHelpers.hpp

```diff
@@ -7,18 +7,23 @@
 #include "DataStructures/Tensor/Expressions/TensorExpression.hpp"
 #include "DataStructures/Tensor/Tensor.hpp"
 
+namespace TestHelpers {
```

Comment:
clang-tidy will want you to do `TestHelpers::TensorExpressions`. This type of namespace scoping is a C++17 feature :)